infra/nnfw/cmake/packages/NPUEngineConfig.cmake

```diff
@@ -0,0 +1,41 @@
+# Looking for pre-installed NPU engine package
```

Comment:
NPUEngine vs TRIXEngine ? Which one is better for this ?

Reply:
Let's change name to TRIXEngine